versions.tf

```diff
@@ -19,16 +19,12 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 5.7.0, <= 5.39.1"
```

Comment:
I think we should allow newer versions of the provider as well. Maybe something like:

```hcl
version = ">= 5.7.0, < 6.0"
```